test/AWS.Deploy.Orchestration.UnitTests/ElasticBeanstalkHandlerTests.cs

```diff
@@ -173,5 +176,93 @@ private bool IsEqual(ConfigurationOptionSetting expected, ConfigurationOptionSet
                 && string.Equals(expected.Namespace, actual.Namespace)
                 && string.Equals(expected.Value, actual.Value);
         }
+
+        [Fact]
+        public async Task SetupWindowsDeploymentManifestTest()
```

Comment:
nit: It's tough to make out what these tests are actually testing, and there's no context in the PR description about testing strategy.

Please add comments on the test methods and throughout that answer something like _"Tests that method X does Y"_.

The second new test looks like it's asserting the same thing as the first. How is it different? If we're testing that an existing file is updated correctly, where are the existing file's contents?

Reply:
I added some comments